doc-bridgepoint/notes/7739_BridgePoint_ClassLoader/7739_BridgePoint_ClassLoader.int.md

```diff
@@ -0,0 +1,75 @@
+---
+
+This work is licensed under the Creative Commons CC0 License
+
+---
+
+# Custom Class Loader
+### xtUML Project Implementation Note
+
+
+1. Abstract
+-----------
+BridgePoint Verifier uses a custom Class Loader to load classes into the execution
+session. The existing Class Loader accesses internal Eclipse components that are
+no longer available in Eclipse 4.4
+
+2. Document References
+----------------------
+[1] [BridgePoint DEI #7739](https://support.onefact.net/redmine/issues/7739) 
```

Comment:
Add 2 spaces at the end of each of these lines so you get line breaks in the rendered output